src/components/chat/ChatList.vue

```diff
@@ -19,6 +24,7 @@
             :value-unread="formatBalance(contact.totalUnreadValue)"
             :num-unread="contact.totalUnreadMessages"
             :compact="compact"
+            @click="vuexSetActiveChat(contact.address)"
```

Comment:
Why did this need to change?

Reply:
Does seem like a better place to put it though.

Reply:
I added logic for the Forum route (and potentially other, future, non-chat routes) wherein the `state.activeChatAddr` is set to `undefined`. With this logic, it made sense to move the vuex action to ChatList vs ChatListItem. That way I could set the `state.activeChatAddr` value of `undefined` for Forum, and also set the `contact.address` value within the same component.